internal/pkg/template/templates/cicd/pipeline_cfn.yml

```diff
@@ -240,7 +240,7 @@ Resources:
             {{- end}}
             {{- range $action := $stage.PreDeployments }}
             - Name: {{ $action.Name }}
-              Region: {{ $stage.Region }}
```

Comment:
qq: do we still need region for stage?

Reply:
A stage doesn't have a "region" configuration if that answers the question!

Reply:
I meant do we still need `Region` for the struct?

Reply:
ah yes, the other actions (the ones backed by CloudFormation) still need to be connected to the env region and thus need to have `Region: {{ $stg.Region }}`